spec/models/report_spec.rb

```diff
@@ -157,16 +157,14 @@
   end
 
   describe "#next_twelve_financial_quarters" do
-    it "returns an array with the next twelve financial quarters" do
-      travel_to(Date.parse("1 April 2020")) do
-        report = create(:report)
-
-        expect(report.next_twelve_financial_quarters).to eq [
-          [2, 2020], [3, 2020], [4, 2020], [1, 2021],
-          [2, 2021], [3, 2021], [4, 2021], [1, 2022],
-          [2, 2022], [3, 2022], [4, 2022], [1, 2023],
-        ]
-      end
+    it "returns an array with the next twelve financial quarters from the date of the report" do
+      report = travel_to(Date.parse("1 April 2020")) { create(:report) }
```

Comment:
Good point! I saw this test in the previous commit and I didn't notice that the whole test was inside the do block, instead of just the report.